src/toolkit/matl_buy_policy.cc

```diff
@@ -247,6 +327,17 @@ void MatlBuyPolicy::AcceptMatlTrades(
     LGH(INFO3) << "got " << it->second->quantity() << " kg of "
                << it->first.request->commodity()  << std::endl;
     buf_->Push(it->second);
+    // ccap handling
+    if (ccap_ > 0) {
```

Comment:
Above you used the condition `ccap_ != -1`. Do these mean the same thing? If so, can we pick one for clarity/readability?

Reply:
Of we can even make a simple inline function:
`include bool use_cumulative_capacity() { return ccap_ > 0; };`